kind/resource_kind.go

```diff
@@ -43,6 +43,13 @@ func resourceKind() *schema.Resource {
 				ForceNew:    true,
 				Computed:    true,
 			},
+			"wait_for_ready": &schema.Schema{
+				Type:        schema.TypeBool,
+				Description: `Defines wether or not the provider will wait for the control plane to be ready. Defaults to true`,
```

Comment:
Here it says default is `true`, but on the next line default is false.
Which was the intended default value?

Reply:
Indeed, it was meant to be false in the description.